css/card.css

```diff
@@ -0,0 +1,56 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+.grid-container{
+    display: grid;
+    background-color: bisque;
+    padding: 10px;
+    grid-template-columns: auto auto auto ;
+    grid-template-rows: auto auto auto ;
+    align-content: center;
+    justify-content: space-around;
+    row-gap:10px;
+    column-gap: 10px;
+    width: 37%;
+    margin:auto;
+}
+
+.card {
+    position: relative;
+    width: 100px;
+    height: 125px;
+    align-items: center;
+    transform-style: preserve-3d;
+    backface-visibility: hidden;
+}
+
+.face{
+    position: absolute;
+    height: 100%;
+    transform-style: preserve-3d;
+    backface-visibility: hidden;
+    transition: 1s ease-in-out;
```

Comment:
The `transition` property should be in the parend class `card` as the whole card will flip not only the image in it.